projects/libs/core-library-typescript/src/test-kit/fake-service-ipc-client.ts

```diff
@@ -1,4 +1,4 @@
-export function generateServiceIPCClient<T>(service: any): T {
+export function generateFakeServiceIPCClient<T>(service: any): T {
```

Comment:
Why do you call it Fake? It's not Fake. It is a Service IPC Client

Reply:
I renamed it back